sideboard/server.py

```diff
@@ -155,6 +156,19 @@ def check_authentication(cls):
     cherrypy_config[setting] = value
 cherrypy.config.update(cherrypy_config)
 
+# Unsubscribe the default server
+cherrypy.server.unsubscribe()
```

Comment:
I probably need to delve into the cherrypy source code to figure out why we're doing this.  Regardless, it would be useful to add a comment here explaining why.  You have several comments explaining what you're doing, but I find "why" comments to be a lot more useful.  In particular, "Unsubscribe the default server" doesn't tell me anything about the purpose of this code or what it's actually accomplishing.  I'm pretty sure that we were already able to set ``server.socket_host`` and such in our config file, so I'm unclear what this code is doing.